packages/shared/src/components/ProfileMenu.tsx

```diff
@@ -84,7 +84,7 @@ export default function ProfileMenu({
         title: 'Logout',
         buttonProps: {
           icon: <PowerIcon />,
-          onClick: logout,
+          onClick: () => logout('manual logout'),
```

Comment:
Let's make this some enum somewhere, just to have it in case we do new logout methods or whatever

Reply:
Made it into an enum